cmd/brew-test-bot.rb

```diff
@@ -984,7 +984,7 @@ def test_ci_upload(tap)
     # These variables are for Jenkins and Circle CI respectively.
     pr = ENV["UPSTREAM_PULL_REQUEST"] || ENV["CIRCLE_PR_NUMBER"]
     if pr
-      pull_pr = "#{tap.remote}/pull/#{pr}"
+      pull_pr = "#{tap.remote.sub(/\.git$/, "")}/pull/#{pr}"
```

Comment:
True, at least for me. Why would Homebrew support atypical git remotes but not `test-bot`? The pair of #35 and #42 implement this support. I'm fine without out it, just curious why.